themes/oh-my-posh/default.omp.json

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
is this the only way to do it? I am not sure we want to keep a default config of some theme like this. maybe we should download it every time, or just not commit it, or vendor it

Reply:
easy, it supports getting it off github, and the user is expected to export it locally and edit it herself. fixed.